### PR TITLE
feat: create customers on mgmt ui

### PR DESCRIPTION
**Pull request by asdfryan**
Fixes: #### 

Ability to create customers on mgmt ui

## Test Plan

Tested locally: 
https://www.loom.com/share/165e4b3fa6ed483594c0fa68c0342f2b

## Deployment instructions

[Add any special deployment instructions here]

### CODE REVIEW

File: apps/mgmt-ui/src/pages/applications/[applicationId]/index.tsx [hunk 1]
```diff
@@ -46,8 +49,14 @@ export const getServerSideProps: GetServerSideProps = async ({ req, res }) => {
 };
 
 export default function Home() {
-  const { customers = [], isLoading } = useCustomers();
+  const { customers = [], isLoading, mutate } = useCustomers();
   const [mobileOpen, setMobileOpen] = useState(false);
+  const applicationId = useActiveApplicationId();
+
+  const onCreateCustomer = async (customerId: string, name: string, email: string) => {
+    await mutate([...customers, { applicationId, customerId, name, email, connections: [] }], false);
```
Comment (albertyfwu) [inline apps/mgmt-ui/src/pages/applications/[applicationId]/index.tsx:57]: shouldn't we create customer before revalidate?

Comment (asdfryan) [reply apps/mgmt-ui/src/pages/applications/[applicationId]/index.tsx]: sure

File: apps/mgmt-ui/src/hooks/useCustomers.ts [hunk 2]
```diff
@@ -3,11 +3,12 @@ import { CustomerExpandedSafe } from '@supaglue/core/types/customer';
 import { useSWRWithApplication } from './useSWRWithApplication';
 
 export function useCustomers() {
-  const { data, isLoading, error } = useSWRWithApplication<CustomerExpandedSafe[]>('/api/internal/customers');
+  const { data, isLoading, error, ...rest } = useSWRWithApplication<CustomerExpandedSafe[]>('/api/internal/customers');
```
Comment (lucasmarshall) [inline apps/mgmt-ui/src/hooks/useCustomers.ts:6]: Seems like we should just remove `isLa
```suggestion
  const { data, isLoading, error, ...rest } = useSWRWithApplication<CustomerExpandedSafe[]>('/api/internal/customers');
```
```suggestion
  const { data, ...rest } = useSWRWithApplication<CustomerExpandedSafe[]>('/api/internal/customers');
```

File: apps/mgmt-ui/src/hooks/useCustomers.ts [hunk 3]
```diff
@@ -3,11 +3,12 @@ import { CustomerExpandedSafe } from '@supaglue/core/types/customer';
...
 
   return {
     customers: data ? camelcaseKeys(data) : undefined,
     isLoading,
     error,
+    ...rest,
```
Comment (lucasmarshall) [inline apps/mgmt-ui/src/hooks/useCustomers.ts:12]:
```suggestion
    ...rest,
```

**Review by lucasmarshall — APPROVED** [review]

**Pull request merged**